src/containers/App/index.test.js

```diff
@@ -16,9 +33,31 @@ describe('<App />', () => {
     expect(props.requestCategoriesAction).not.toBeUndefined();
   });
 
+  it('should scroll to top on history change', () => {
+    render(
+      withAppContext(<AppContainer requestCategoriesAction={() => { }} />),
+    );
+
+    expect(spyScrollTo).not.toHaveBeenCalled();
+
+    act(() => {
+      history.push('/somewhere/else');
+    });
+
+    act(() => {
```

Comment:
you should test that `spyScrollTo` has not been called after pushing

Reply:
gedaan